docs/pages/components/ClickOutsideListener.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: ClickOutsideListener
+description: A utility component to detect click events outside of a wrapped component.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/ClickOutsideListener/index.tsx
+---
+
+import { ClickOutsideListener, Button } from '@deque/cauldron-react'
+
+```js
+import { ClickOutsideListener } from '@deque/cauldron-react'
+```
+
+## Example
+
+```jsx example
+<ClickOutsideListener onClickOutside={() => alert('You clicked outside!')}>
+  <Button>Click Inside</Button>
+</ClickOutsideListener>
+````
+
+## Props
+
+<ComponentProps
+  children={true}
+  props={[
+    {
+      name: 'onClickOutside',
+      type: ['MouseEvent', 'TouchEvent'],
+      description: 'Callback function for event type.'
```

Comment:
looks like this prop is required but I don't see that explicitly in the new docs

Reply:
Yup, good catch. This should be required.